autosklearn/automl.py

```diff
@@ -386,16 +379,16 @@ def fit(
         requirements = pkg_resources.resource_string('autosklearn', 'requirements.txt')
         requirements = requirements.decode('utf-8')
         requirements = [requirement for requirement in requirements.split('\n')]
-        for requirement in requirements:
-            if not requirement:
-                continue
-            match = RE_PATTERN.match(requirement)
-            if match:
-                name = match.group('name')
-                module_dist = pkg_resources.get_distribution(name)
-                self._logger.debug('  %s', module_dist)
-            else:
-                raise ValueError('Unable to read requirement: %s' % requirement)
+        # for requirement in requirements:
```

Comment:
Same here with the comments :), yeah I know this is a draft, so probably something you already had in mind removing.

Reply:
Yes, same issue as above. Maybe I should just update the dependency parser to ignore the version in such cases.